docs/content/maintenance/metrics.md

```diff
@@ -0,0 +1,387 @@
+            <td>Bucket</td>
+            <td>Gauge</td>
+            <td>Number of changelog files compacted in last compaction.</td>
+        </tr>
+        <tr>
+            <td>lastRewriteInputFileSize</td>
+            <td>Bucket</td>
+            <td>Gauge</td>
+            <td>Size of deleted files in the last compaction.</td>
+        </tr>
+        <tr>
+            <td>lastRewriteOutputFileSize</td>
+            <td>Bucket</td>
+            <td>Gauge</td>
+            <td>Size of added files in the last compaction.</td>
+        </tr>
+        <tr>
+            <td>lastRewriteChangelogFileSize</td>
+            <td>Bucket</td>
+            <td>Gauge</td>
+            <td>Size of changelog files compacted in last compaction.</td>
+        </tr>
+    </tbody>
+</table>
+
+## Source Metrics (Flink)
+
+<table class="table table-bordered">
+    <thead>
+    <tr>
+      <th class="text-left" style="width: 225pt">Metrics Name</th>
+      <th class="text-left" style="width: 65pt">Level</th>
+      <th class="text-left" style="width: 70pt">Type</th>
+      <th class="text-left" style="width: 300pt">Description</th>
+    </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>currentFetchEventTimeLag</td>
+            <td>Flink Source Operator</td>
+            <td>Gauge</td>
+            <td>Time difference between reading the data file and file creation.</td>
+        </tr>    
+    </tbody>
+</table>
+
+## Sink Metrics (Flink)
+
+<table class="table table-bordered">
+    <thead>
+    <tr>
+      <th class="text-left" style="width: 225pt">Metrics Name</th>
+      <th class="text-left" style="width: 65pt">Level</th>
+      <th class="text-left" style="width: 70pt">Type</th>
+      <th class="text-left" style="width: 300pt">Description</th>
+    </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>numBytesOut</td>
+            <td>Table</td>
+            <td>Counter</td>
+            <td>The total number of output bytes.</td>
+        </tr>
+        <tr>
+            <td>numBytesOutPerSecond</td>
+            <td>Table</td>
+            <td>Meter</td>
+            <td>The output bytes per second.</td>
+        </tr>
+        <tr>
+            <td>numRecordsOut</td>
+            <td>Table</td>
+            <td>Counter</td>
+            <td>The total number of output records.</td>
+        </tr>
+        <tr>
+            <td>numRecordsOutPerSecond</td>
+            <td>Table</td>
+            <td>Meter</td>
+            <td>The output records per second.</td>
+        </tr>  
+    </tbody>
+</table>
+
+# Bridging To Flink
+
+Paimon has implemented bridging metrics to Flink's metrics system, which can be reported by Flink, and the lifecycle of metric groups are managed by Flink.
+
+Please join the `<scope>.<infix>.<metric_name>` to get the complete metric identifier when using Flink to access Paimon, `metric_name` can be got from [Metric List]({{< ref "maintenance/metrics#metrics-list" >}}).
```

Comment:
Provide an example here. "For example, the identifier of metric `lastScanDuration` for table `MyTable` is xxxxxx".